### ISSUE TITLE
do i need to move away form this project?

### ISSUE THREAD

**Issue by hs6666** [airsonic-advanced/airsonic-advanced#954]
Hi,

Hmm... no updates for a ages, runs on java, mine is exposed to the internet :(

is there any alternative out there as good as this?

If not, I suppose only alterative would be to remove it or close the exposed ports and put it behind a VPN

Probably go for the latter since the software is awesome :)

**Comment by langfingaz** [comment]
Would you mind changing the title to something like "Maintenance / Project status"? I think that's more appropriate, thanks!

As there is just one commit since end of march, this project does indeed seem unmaintained. Can @airsonic-advanced please give a short message about the current and/or future status? Thanks!

**Comment by langfingaz** [comment]
@hs6666 Besides using a VPN you could also set up basic authentication in a webserver (e.g. nginx) that forwards the reqests to Airsonic Advanced. This way one has to authenticate (login) at your up-to-date webserver before one can access Airsonic.

**Comment by hs6666** [comment]
i've just discovered zerotier and installed it on my airsonic server and phone etc and closed the port. the simplest easiest vpn i think i've ever setup :)

**Comment by Trufax** [comment]
Using a VPN is a short term fix but in general you should nut run dead software. I hope its just a slowdown of the project development or a break. Subsonic died, Airsonic died, lets hope Airsonic Advanced stays alive. =)

**Comment by hs6666** [comment]
I always start with the assumption that all projects will eventually die if they are not supported by a parallel revenue "business" stream, its natural/normal.

Peoples priorites/commitments change with time/age and with it the time/energy they can devote to it, especially wrt to personal financial needs/changes

**Comment by khers** [comment]
The parallel revenue stream didn't save Subsonic...

Are there any contributors here willing to pick up the project if it is dead? I have never done Java development and hope to keep it that way...

**Comment by khers** [comment]
AFAIC Jun 25 was the last time the maintainer of this project was active on GitHub, I hope they are okay.

**Comment by hs6666** [comment]
Thats what I was thinking with the sudden drop off :(

Unfortunantly most new business's fail with the first two years, it's a hard game!

Love to help in this project, but my coding skills go no further then amending others (simple) python code to utilize in my home lab

**Comment by yllekz** [comment]
Would also really like to know what's going on with this project. Things are starting to break and need attention.

**Comment by monik3r** [comment]
It seems like there has been some dev work going on in https://github.com/kagemomiji/airsonic-advanced. Unsure what exactly the dev is doing (I'm not super familiar with the code), but by the number of commits it seems like a good bit of effort.

**Comment by hs6666** [comment]
intereting, hope it gains tractions, have just subscribed to kagemomiji github notifications :)

Personally, at the moment, I have a "closed" system with tailscale running on my server and on my phone and have no performance/technical issues with this "old" build for day to day use so far....

**Comment by 4oo4** [comment]
I'm testing @kagemomiji's fork now. If you're reading this, it would be awesome to have a changelog of everything you've added since forking, I read a few of the commits and it looks like a lot of vulns are getting fixed, but it would be awesome to have a full list since you've done so much. Also thanks for all the work you're putting in! I'm really happy to see that trivy scans have been added to CI.

**Comment by kagemomiji** [comment]
@4oo4 
I wrote [CHANGELOG](https://github.com/kagemomiji/airsonic-advanced/blob/main/CHANGELOG.md).
If you want to know my changes, please see changelogs of v11.0.1 and v11.0.2.

**Comment by DutchComputerKid** [comment]
Starred @kagemomji's work, very nice!
Development of all these versions and forks  has gone all over the place, there's kage's fork, JPSonic's fork and whatever else is still out there. 
Got those two on testing right now (plenty of problems) and the original Subsonic still running (Albeit Subsonic has its activation cracked and still rock-stable).

I dont suppose there would be a single place for now for what would become the de-facto modern version anytime soon?

**Comment by monik3r** [comment]
Something that might be good is to reach out to each active developer and see what their intention is for their repo. It takes a good bit of work to engage with users and maintain a project, so it might be good to make sure they want to do that.

**Comment by litebito** [comment]
> Something that might be good is to reach out to each active developer and see what their intention is for their repo. It takes a good bit of work to engage with users and maintain a project, so it might be good to make sure they want to do that.

That would be a good idea indeed. Airsonic-Advanced is still one of the best open source streaming servers out here

**Comment by randomnicode** [comment]
@kagemomiji want to open prs so we can integrate?

**Comment by kagemomiji** [comment]
@randomnicode 

OK thanks. 
I will divide my works to some PRs. Please wait.

**Comment by kagemomiji** [comment]
@randomnicode 
Perhaps I misunderstood? Do you mean that you want to integrate into my forked repository?

**Comment by bartmans** [comment]
I've been using Kagemomiji's fork for a while now. Two bugs I submitted were fixed fast by him, Otherwise it runs smooth. So for now I'm switching. Hope development picks up even more with an active repo.

**Comment by Trufax** [comment]
I also moved to Kagemomiji's fork and It works fine.

**Comment by checketts** [comment]
@kagemomiji @randomnicode I also began upgrading to the latest Spring Boot (and in prep for other PRs and improvements)

I would love to contribute back without causing further forks. Can I help create a PR of @kagemomiji 's repo back into this repo?

**Comment by kagemomiji** [comment]
@checketts 
Of course. Your help would be greatly appreciated. However, the cooperation of @randomnicode is essential, although he seems to be not very active recently.
